pkgs/development/python-modules/django-filter/default.nix

```diff
@@ -1,32 +1,32 @@
 { lib
 , buildPythonPackage
 , fetchPypi
+, flit-core
 , django
 , djangorestframework, python, mock
 }:
 
 buildPythonPackage rec {
   pname = "django-filter";
-  version = "23.2";
+  version = "23.3";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-L+FfeBCEde2lJWkoEyBfpvnowcrxrmXapYYtQDxtvwA=";
+    hash = "sha256-AV/hVVguGAW0Bik0Tkps88xARQgn0pTQQLS4wXSan6Y=";
   };
 
+  format = "pyproject";
+
+  nativeBuildInputs = [ flit-core ];
+
   propagatedBuildInputs = [ django ];
 
   pythonImportsCheck = [
     "django_filters"
   ];
 
-  # Tests fail (needs the 'crispy_forms' module not packaged on nixos)
-  doCheck = false;
-
   nativeCheckInputs = [
     djangorestframework
-    django
-    mock
   ];
 
   checkPhase = ''
```

Comment:
the homepage is https://github.com/carltongibson/django-filter
the changelog is here `https://github.com/carltongibson/django-filter/blob/v${version}/CHANGES.rst`

Reply:
Done